src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -108,25 +91,25 @@ Vec3f D_808973A4[] = {
     { 0.0f, 101.4f, 5.0f },
 };
 
-Vec3f D_808974DC[] = {
+static Vec3f D_808974DC[] = {
     { 12.0f, 21.300001f, -2.5f },  { 30.0f, 21.300001f, -2.5f }, { -15.0f, 21.300001f, -2.5f },
     { -30.0f, 21.300001f, -2.5f }, { 12.0f, 21.300001f, -2.5f },
 };
 
-s32 D_80897518[] = { 0x80, 0xA0, 0xA0, 0x80 };
+static s32 D_80897518[] = { 0x80, 0xA0, 0xA0, 0x80 };
 
-InitChainEntry D_80897528[] = {
+static InitChainEntry D_80897528[] = {
```

Comment:
```suggestion
static InitChainEntry sInitChain[] = {
```

Reply:
I'm going to leave these for a documentation pass. This is just for matching.

Reply:
eh, I'll do it anyways.